### DISCUSSION TITLE
CVE-2023-38831 winrar 漏洞分析 - B1TG

### DISCUSSION THREAD

**Discussion by giscus** [b1tg/b1tg.github.io#3] [Discuss]
# CVE-2023-38831 winrar 漏洞分析 - B1TG

CVE-2023-38831 winrar 漏洞分析

https://b1tg.github.io/post/cve-2023-38831-winrar-analysis/

**Comment by miffi-oxo** [comment]
您好，我们正在参考您写的文章分析CVE-2023-38831的弱点。 但是在调试器上搜索了PathFileExistsDefExtAndAttributesW函数，但没有出来。 即使订阅了有关这些信息，也很难获得有意义的信息。 可以问一下如何找到该函数，如何分析吗？

**Reply by b1tg** [reply]
It exists in shlwapi.dll on Windows 7, newer shlwapi.dll have different implement but the behavior is same.

**Reply by miffi-oxo** [reply]
Thankfully, I was able to find it thanks to your help. Thank you.
